src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -9413,6 +9413,158 @@ void LF()
             Assert.False(comp.HasLocalsInit("C.<>c.<M>b__0_5")); // LL
         }
 
+        [Fact]
+        public void SkipLocalsInit_LocalFunctionAttribute()
+        {
+            var source = @"
+namespace System.Runtime.CompilerServices
+{
+    public class SkipLocalsInitAttribute : System.Attribute
+    {
+    }
+}
+
+public class C
+{
+    public void M()
+    {
+        [System.Runtime.CompilerServices.SkipLocalsInitAttribute]
+        void F()
+        {
+            int y = 1;
+            y = y + y + y;
+
+            void FF()
+            {
+                int x = 2;
+                x = x + x + x;
+            }
+
+            System.Action FL = () =>
+            {
+                int x = 3;
+                x = x + x + x;
+            };
+        }
+
+        System.Action L = () =>
+        {
+            int y = 4;
+            y = y + y + y;
+
+            void LF()
```

Comment:
>void LF() [](start = 12, length = 9)

It looks like we are missing a positive test for a scenario when a local function is nested into a lambda.